src/__tests__/findByApi.test.js

```diff
@@ -18,19 +18,19 @@ test('findBy queries work asynchronously', async () => {
   } = render(<View />);
   await expect(findByTestId('aTestId', options)).rejects.toBeTruthy();
   await expect(findAllByTestId('aTestId', options)).rejects.toBeTruthy();
-  await expect(findByText('Some Text', options)).rejects.toBeTruthy();
-  await expect(findAllByText('Some Text', options)).rejects.toBeTruthy();
+  await expect(findByText('Some Text', {}, options)).rejects.toBeTruthy();
+  await expect(findAllByText('Some Text', {}, options)).rejects.toBeTruthy();
   await expect(
-    findByPlaceholderText('Placeholder Text', options)
+    findByPlaceholderText('Placeholder Text', {}, options)
```

Comment:
this is the second breaking change I mentioned

Reply:
We could provide an easy migration path by looking for `waitFor` options (`timeout`, `interval`) also in the `queryOptions` only for `findBy` queries and only till next major release. While displaying a console warning to inform the developer. Wdyt?

Reply:
That's a good idea that I missed, I fixed it in the latest commit d2f9f337faa45f945c82bcaf41fc8145e00403b6